src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -143,7 +143,7 @@ private function addRandomTestsOrderAttributesIfNotSet(string $version, SafeDOMX
             return;
         }
 
-        if ($this->addAttributeIfNotSet('executionOrder', 'random', $xPath)) {
+        if ($this->addAttributeIfNotSet('executionOrder', 'defects,random', $xPath)) {
```

Comment:
if I understand it correctly, Infection's config validation for project's config will not be broken as we validate a bit earlier, right?